book/09-git-and-other-scms/sections/client-p4.asc

```diff
@@ -442,10 +442,10 @@ Running it drops us into our favorite editor, and the contents of the file look
 #  Type:        Either 'public' or 'restricted'. Default is 'public'.
 #  Description: Comments about the changelist.  Required.
 #  Jobs:        What opened jobs are to be closed by this changelist.
-#               You may delete jobs from this list.  (New changelists only.)
+#               You may delete jobs from this list (new changelists only).
 #  Files:       What opened files from the default changelist are to be added
-#               to this changelist.  You may delete files from this list.
-#               (New changelists only.)
+#               to this changelist. You may delete files from this list
+#               (new changelists only).
```

Comment:
It doesn't makes sense to "fix" this, since it's verbatim output from a program we didn't write.

Reply:
>  it's verbatim output from a program

Sorry, I missed that. It seems like using period before parentheses is very common, I've seen this in git(1) man page and other places too. Is it a right thing to do (move period after parentheses)?